lib/ansible/modules/remote_management/redfish/redfish_config.py

```diff
@@ -0,0 +1,233 @@
+  command:
+    required: true
+    description:
+      - List of commands to execute on OOB controller
+  baseuri:
+    required: true
+    description:
+      - Base URI of OOB controller
+  user:
+    required: true
+    description:
+      - User for authentication with OOB controller
+  password:
+    required: true
+    description:
+      - Password for authentication with OOB controller
+  bios_attr_name:
+    required: false
+    description:
+      - name of BIOS attribute to update
+  bios_attr_value:
+    required: false
+    description:
+      - value of BIOS attribute to update
+  mgr_attr_name:
+    required: false
+    description:
+      - name of Manager attribute to update
+  mgr_attr_value:
+    required: false
+    description:
+      - value of Manager attribute to update
+
+author: "Jose Delarosa (github: jose-delarosa)"
+'''
+
+EXAMPLES = '''
+  - name: Set BootMode to UEFI
+    redfish_config:
+      category: Systems
+      command: SetBiosAttributes
+      bios_attr_name: BootMode
+      bios_attr_value: Uefi
+      baseuri: "{{ baseuri }}"
+      user: "{{ user }}"
+      password: "{{ password }}"
+
+  - name: Set BootMode to Legacy BIOS
+    redfish_config:
+      category: Systems
+      command: SetBiosAttributes
+      bios_attr_name: BootMode
+      bios_attr_value: Bios
+      baseuri: "{{ baseuri }}"
+      user: "{{ user }}"
+      password: "{{ password }}"
+
+  - name: Enable PXE Boot for NIC1
+    redfish_config:
+      category: Systems
+      command: SetBiosAttributes
+      bios_attr_name: PxeDev1EnDis
+      bios_attr_value: Enabled
+      baseuri: "{{ baseuri }}"
+      user: "{{ user }}"
+      password: "{{ password }}"
+
+  - name: Set BIOS default settings
+    redfish_config:
+      category: Systems
+      command: SetBiosDefaultSettings
+      baseuri: "{{ baseuri }}"
+      user: "{{ user }}"
+      password: "{{ password }}"
+
+  - name: Enable NTP in the OOB Controller
+    redfish_config:
+      category: Manager
+      command: SetManagerAttributes
+      mgr_attr_name: NTPConfigGroup.1.NTPEnable
+      mgr_attr_value: Enabled
+      baseuri: "{{ baseuri }}"
+      user: "{{ user}}"
+      password: "{{ password }}"
+
+  - name: Set NTP server 1 to {{ ntpserver1 }} in the OOB Controller
+    redfish_config:
+      category: Manager
+      command: SetManagerAttributes
+      mgr_attr_name: NTPConfigGroup.1.NTP1
+      mgr_attr_value: "{{ ntpserver1 }}"
+      baseuri: "{{ baseuri }}"
+      user: "{{ user}}"
+      password: "{{ password }}"
+
+  - name: Set Timezone to {{ timezone }} in the OOB Controller
+    redfish_config:
+      category: Manager
+      command: SetManagerAttributes
+      mgr_attr_name: Time.1.Timezone
+      mgr_attr_value: "{{ timezone }}"
+      baseuri: "{{ baseuri }}"
+      user: "{{ user}}"
+      password: "{{ password }}"
+'''
+
+RETURN = '''
+result:
+    description: different results depending on task
+    returned: always
+    type: dict
```

Comment:
Maybe consider changing the type to `complex` and documenting a few different examples of data returned by the commands. Since there are only three, this should be feasible unless it's just an enormous amount of data that is returned. See [docs](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block) for examples.

Reply:
Need to update this, this was true before I split my original module into 3 modules. In this module, (redfish_config) the return value is either OK (success) or an error message/code.